cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -498,9 +498,30 @@
 #include "umm_malloc.h"
 
 #include "umm_malloc_cfg.h"   /* user-dependent */
+#if defined(DEBUG_ESP_PORT)
```

Comment:
This is specific to the ESP. It would not be accepted into umm upstream. I suggest moving this elsewhere, e.g. the config file.

Reply:
As described later this can be deleted.